### ISSUE TITLE
rpmlib-sys: rpmts.h and rpmfi.h cause `bindgen_test_layout_max_align_t` failures

### ISSUE THREAD

**Issue by tony-iqlusion** [iqlusion-io/crates#11]
These headers are:

* `rpmts.h`: Transaction sets
* `rpmfi.h`: File information

When either of these headers are enabled (in `src/rpmlib-sys.h`), they cause the automatically generated `bindgen_test_layout_max_align_t` test to fail. In either case, we get the following failure:

```
---- bindgen_test_layout_max_align_t stdout ----
	thread 'bindgen_test_layout_max_align_t' panicked at 'assertion failed: `(left == right)`
  left: `24`,
 right: `32`: Size of: max_align_t', target/debug/build/rpmlib-sys-25076a13bc63918f/out/bindings.rs:6564:5
```

Both of these headers are presently commented out in the binding generator due to these errors.

**Comment by tony-iqlusion** [comment]
Transaction sets are an integral part of rpmlib's API, and without them we're blocked on doing pretty much anything (#13, #14, #15, #16).

**Comment by tony-iqlusion** [comment]
These were both fixed in #17 by having bindgen build a C++ binding instead of a C one

**Comment by tarcieri** [comment]
Per https://github.com/rust-lang-nursery/rust-bindgen/issues/363 it sounds like this issue would've also been addressed by using `rust_target(1.25)` now that `repr(align(N))` is stable.

**Comment by tony-iqlusion** [comment]
Per https://github.com/rust-lang-nursery/rust-bindgen/issues/363 it sounds like this issue would've also been addressed by using `rust_target(1.25)` now that `repr(align(N))` is stable.
